sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -827,6 +829,12 @@ def dynamic_queries(self):
             else:
                 self.log_missing_metric("Failover Cluster Instance", major_version, engine_edition)
 
+        if is_affirmative(self.instance.get('include_log_shipping_primary', False)):
```

Comment:
Should we add a warning when both primary and secondary options are set to `True`?

Reply:
I think that _TECHNICALLY_ you could have log shipping from A to B and from B to C (on different DBs).

Reply:
Ahh ok. So the primary vs. secondary is relative. A db could be both primary and secondary.

Reply:
A sqlserver instance yes, but a DB is either primary or secondary. So a given instance could have two DBs, let's say `foo` and `bar`, and `foo` could be log shipping to host M, while `bar` could be _log-shipped FROM_ host N. So in this instance, `foo` is a primary and `bar` is a secondary.